crates/bevy_render/src/mesh/mesh/mod.rs

```diff
@@ -49,8 +49,20 @@ pub struct Mesh {
 /// # use bevy_render::render_resource::PrimitiveTopology;
 /// fn create_triangle() -> Mesh {
 ///     let mut mesh = Mesh::new(PrimitiveTopology::TriangleList);
-///     mesh.insert_attribute(Mesh::ATTRIBUTE_POSITION, vec![[1.0, 0.0, 0.0], [0.0, 1.0, 0.0], [1.0, 1.0, 0.0]]);
-///     mesh.set_indices(Some(Indices::U32(vec![0,1,2])));
+///     mesh.insert_attribute(
+///         Mesh::ATTRIBUTE_POSITION,
+///         vec![[1.0, 0.0, 0.0], [0.0, 1.0, 0.0], [1.0, 1.0, 0.0], [0.0, 0.0, 0.0]]
+///     );
+///     // We have created 4 vertices, each with it's own position attribute.
+///     mesh.insert_attribute(
+///         Mesh::ATTRIBUTE_UV_0,
+///         vec![[0.0, 0.0], [1.0, 0.0], [0.0, 1.0], [1.0, 1.0]]
+///     );
+///     // We have assigned a UV coordinate for each of the 4 vertices we created.
+///     // NOTE: Multiple vertices can have the same attribute, but multiple attributes cannot be
+///     // of the same vertex.
```

Comment:
This "Note" isn't clear to me, what do you mean by that?

Reply:
This comment was meant to make sure people understand they cannot assign multiple attributes of the same kind to the same vertex, and there is not problem with multiple vertices having the same position, or the same UV coordinate, in fact it's common practice in complex 3D modeling. I will try to rephrase, or provide a small example (but that may be pushing it on the length of a short docs example), or remove it.

Reply:
Okay, good thing to mention yes!

You can also explain that outside of the example.